### PR TITLE
Mini/label creation

### PR DESCRIPTION
**Pull request by mini102**

### CODE REVIEW

File: Assets/Scripts/baseGridManger.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
+using System;
```
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:1]: 스크립트 이름은 baseGridManger보다는 getSimilarity같은 이름은 어떠실까요?

File: Assets/Scripts/baseGridManger.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    {
+        setLabelImage(filePath);
+        drawing = board.GetComponent<GridManger>();
+    }
+
+    void Update(){
+        loss = similarity();
+    }
+
+    float abs(float x){
```
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:35]: 메서드 이름은 대문자로 시작해야합니다.

File: Assets/Scripts/baseGridManger.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        if(x<0) x*=-1;
+        return x;
+    }
+
+    float similarity(){
```
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:40]: 메서드 이름은 대문자로 시작해야합니다.

File: Assets/Scripts/baseGridManger.cs (hunk 1 above)
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:1]: How about the name of the script like 'getSimilarity' instead of 'baseGridManager'?

File: Assets/Scripts/baseGridManger.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System.Linq;
+using System.IO;
+
+public class baseGridManger : MonoBehaviour
```
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:9]: The class and method name must start with an uppercase letter. Please change everything.

File: Assets/Scripts/baseGridManger.cs [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+{
+    //public PixelColor pixelObject;
+    public GameObject board;
+
+    public int gridSize = 40;
+    
+    private Color[][] colorArray;
+    public string filePath;
+    //texture 씌울 오브젝트
```
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:18]: Please change all comments to English.

File: Assets/Scripts/baseGridManger.cs [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+    void setLabelImage(string filePath)
+    {
+        Texture2D tex = new Texture2D(gridSize, gridSize);
+        byte[] bytes = File.ReadAllBytes(filePath);
+        tex.LoadImage(bytes);
+
+        Texture2D result = resizing(tex,gridSize);
+
+        //Color[] label = tex.GetPixels(0,0,gridSize,gridSize);
```
Comment (dkdud7622) [inline Assets/Scripts/baseGridManger.cs:78]: Please delete unused comment.

Comment (dkdud7622) [reply Assets/Scripts/baseGridManger.cs:78]: Please delete all of them before sending the PR request.

File: Assets/Scripts/GridManger.cs [hunk 7]
```diff
@@ -141,11 +142,25 @@ public void SaveColorArrayAsImage()
         int rand = r.Next();
         File.WriteAllBytes(dirPath + "/" + rand + ".png", bytes);
     }
+
+    public Color getColor(int x,int y){
```
Comment (dkdud7622) [inline Assets/Scripts/GridManger.cs:146]: The class and method name must start with an uppercase letter. Please change everything.

**Review by dkdud7622 — CHANGES_REQUESTED** [review]
